machine.go

```diff
@@ -374,7 +378,12 @@ func (m *Machine) startup(command string, extracontent [][2]string) int {
 	w.WriteFile("/wrapper",
 		fmt.Sprintf(commandWrapper, command), 0755)
 
-	w.WriteFile("/init", initScript, 0755)
+	shebang := "#!/bin/busybox sh"
+	if mergedUsrSystem() {
+		shebang = "#!/usr/bin/busybox sh"
+	}
+	w.WriteFile("/init",
+		fmt.Sprintf("%s\n%s", shebang, initScript), 0755)
```

Comment:
Why this complexity, on a merged usr system /bin is a symlink to /usr/bin, so #!/bin/busybox should work in all cases afaik